packages/terra-action-footer/src/_ActionFooterContainer.jsx

```diff
@@ -8,21 +8,25 @@ const cx = classNames.bind(styles);
 
 const propTypes = {
   children: PropTypes.node,
+  isBorderless: PropTypes.bool,
```

Comment:
Lets add a prop description here. Even though its an internal component, its nice to have quick reference to props intention in the component itself.